src/components/Birthday.tsx

```diff
@@ -0,0 +1,29 @@
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import {useState} from "react";
+import "./birthday.scss";
+
+const Birthday = () => {
+  const [value, setValue] = useState<any>(null);
+  const fieldHandler = (event: Date) => {
```

Comment:
it is better to name the argument here as date (not event)

const fieldHandler = (date: Date) => {